erpnext/hr/doctype/employee/employee.js

```diff
@@ -36,6 +41,15 @@ erpnext.hr.EmployeeController = frappe.ui.form.Controller.extend({
 	},
 
 });
+function total_previous_experience(doc){
+	var total_months = 0;
+	for(var i=0;i< doc.external_work_history.length;i++){
+		var row = doc.external_work_history[i];
+		total_months += parseFloat(row.total_experience_months);
+	}
+	cur_frm.set_value("total_previous_experience_months", total_months);
```

Comment:
`cur_frm` is deprecated.

https://github.com/frappe/erpnext/wiki/Coding-Standards